R/utils.R

```diff
@@ -104,3 +104,22 @@ get_teal_bs_theme <- function() {
     bs_theme
   }
 }
+
+#' Include `JS` files from `/inst/js/` package directory to application header
+#'
+#' `system.file` should not be used to access files in other packages, it does
+#' not work with `devtools`. Therefore, we redefine this method in each package
+#' as needed. Thus, we do not export this method
+#'
+#' @param pattern (`character`) pattern of files to be included, passed to `system.file`
+#' @param except (`character`) vector of basename filenames to be excluded
+#'
+#' @return HTML code that includes `JS` files
+#' @keywords internal
+include_js_files <- function(pattern = NULL, except = NULL) {
+  checkmate::assert_character(except, min.len = 1, any.missing = FALSE, null.ok = TRUE)
```

Comment:
`min.len = 1` precludes NULL

Reply:
I've removed exclude at all as it's not used in this package. However, `min.len = 1` and `null.ok = TRUE` can be specified together which prevents someone from specifying `character(0)`

Reply:
But if `min.len = 1`, NULL will not pass as it has length 0. `min.len = 1` is sufficient here.

Reply:
```r
checkmate::assert_character(NULL, min.len = 1, any.missing = FALSE, null.ok = TRUE)
#> silent
```

Reply:
I stand corrected.